packages/core/src/base-axis-chart.ts

```diff
@@ -463,7 +481,7 @@ export class BaseAxisChart extends BaseChart {
 		yGrid.ticks(scales.y.numberOfTicks || Configuration.scales.y.numberOfTicks);
 
 		const g = this.innerWrap.select(".y.grid")
-			.attr("transform", "translate(0, 0)")
+			.attr("transform", `translate(0, 0)`)
```

Comment:
These could stay as normal quotes
```suggestion
			.attr("transform", "translate(0, 0)")
```